sw/airborne/firmwares/rotorcraft/autopilot_utils.c

```diff
@@ -130,9 +130,8 @@ void WEAK set_rotorcraft_commands(pprz_t *cmd_out, int32_t *cmd_in, bool in_flig
     cmd_in[COMMAND_THRUST] = 0;
   }
 #endif
-  cmd_out[COMMAND_ROLL] = cmd_in[COMMAND_ROLL];
-  cmd_out[COMMAND_PITCH] = cmd_in[COMMAND_PITCH];
-  cmd_out[COMMAND_YAW] = cmd_in[COMMAND_YAW];
-  cmd_out[COMMAND_THRUST] = cmd_in[COMMAND_THRUST];
+  for (int i=0; i < COMMANDS_NB; i++) {
+    cmd_out[i] = cmd_in[i];
+  }
```

Comment:
This should be moved to the top of the function, otherwise the yaw and thrust commands might not be correct.

Reply:
I don't think I agree: the safety is changing ```cmd_in``` and you want the safe value to be copied to the commands.

Reply:
Hum, right. Modifying the input is maybe questionable thought...

Reply:
This would overwrite the CommandsFromRc that is done at a different frequency so I reverted this.